docs/source/new-project.rst

```diff
@@ -481,6 +482,10 @@ Once your training starts, you can view the logs by using your favorite logger o
 
     tensorboard --logdir ./lightning_logs
 
+.. note::
+    Lightning automatically shows the loss value returned from ``training_step`` in the progress bar.
+    So, no need to explicitly log like this ``self.log('loss', loss, prog_bar=True)``.
+
```

Comment:
Can someone confirm this is true?
Related: https://github.com/PyTorchLightning/pytorch-lightning/issues/3756

Reply:
This is true. Take for example out basic [mnist](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pl_examples/basic_examples/mnist.py) example. `self.log` is not called during `training_step` but the loss returned is still showing during training if you run the script.

Reply:
@SkafteNicki Thank you for confirming